src/main/fc/controlrate_profile.c

```diff
@@ -40,14 +40,20 @@ void pgResetFn_controlRateProfiles(controlRateConfig_t *instance)
     for (int i = 0; i < MAX_CONTROL_RATE_PROFILE_COUNT; i++) {
         RESET_CONFIG(controlRateConfig_t, &instance[i],
             .rcExpo8 = 70,
+            .manual_pitch_rcExpo8 = 70,
+            .manual_roll_rcExpo8 = 70,
```

Comment:
Also, could you add those new setting to MSP? Either MSP_FW_CONFIG but even better as separate MPS2 frame. We have plenty of IDs over there. 
And add frame description to https://github.com/iNavFlight/inav/wiki/INAV-MSP-frames-changelog

Reply:
Yes sure good idea. I didn't think about it.

Reply:
@DzikuVx Are you sure there is something to change to be able to get/set the new settings with MSP2 ? It is the first time I read the code but from what I understand any settings can be get/set with MSP2_COMMON_SETTING/MSP2_COMMON_SET_SETTING frame types.

Reply:
I gotta agree with @shellixyz on this one. I'd rather NOT see these settings in dedicated MSP commands. They don't seem to fit the use case of being frequently polled and or updated during a flight, so the overhead of MSP2_COMMON_SETTING/MSP2_COMMON_SET_SETTING should make no performance difference. It will, however, save a bit of flash and a bunch of developer time (which is the scarcest resource of them all!).